tests/Mutator/ReturnValue/AbstractValueToNullReturnValueTest.php

```diff
@@ -137,26 +171,33 @@ public function test_it_does_not_mutate_when_function_contains_another_function_
     {
```

Comment:
Actually I'm not sure about this. Could we double check that if you have:

```php
class Test
{
    function __invoke($x): void {
        $foo = function ($y): bool {
            return true;
        }

        return $foo($x);
    }
}
```

Then _maybe_ the mutants changing the `return true` to `return false` does work. I think the issue with function is only named functions not anonymous functions like that. If that's the case then even the following should work:

```php
$x = function foo() {
    return true; 
}

// As opposed of:

function foo() {
    // the content of this function (namespaced or not) cannot not be mutated
}
```

Reply:
Actually, this is not a valid syntax. Anonymous function can't have a name.
```php
$x = function foo() {
    return true; 
}
```

I checked your example in this branch and there is a correct mutant:

```diff
class Test
{
    function __invoke($x): void {
        $foo = function ($y): bool {
-            return true;
+            return false;
        }

        return $foo($x);
    }
}
```